features/src/main/feature/camel-features.xml

```diff
@@ -1907,16 +1907,20 @@
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-stream/${project.version}</bundle>
     </feature>
-<!--    <feature name='camel-stomp' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${camel.osgi.version.range}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>mvn:org.apache.geronimo.specs/geronimo-jta_1.1_spec/${geronimo-jta-spec-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.apache.geronimo.specs/geronimo-jms_1.1_spec/${geronimo-jms-spec-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.fusesource.stompjms/stompjms-client/${stompjms-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.fusesource.hawtbuf/hawtbuf/${hawtbuf-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.fusesource.hawtdispatch/hawtdispatch/${hawtdispatch-version}</bundle>-->
-<!--        <bundle dependency='true'>mvn:org.fusesource.hawtdispatch/hawtdispatch-transport/${hawtdispatch-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-stomp/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-stomp' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <details>
+            The bundle wrap:mvn:org.apache.geronimo.specs/geronimo-jms_1.1_spec/1.1 provides javax.jms package
+            required by mvn:org.fusesource.stompjms/stompjms-client/${stompjms-version} bundle. The wrap protocol is used because
+            it is not a valid OSGI bundle.
+        </details>
+        <bundle dependency='true'>wrap:mvn:org.apache.geronimo.specs/geronimo-jms_1.1_spec/1.1</bundle>
```

Comment:
Had to wrap this bundle because the feature build fails, and upon installing it on Karaf the following error occurs
`Unable to install bundle mvn:org.apache.geronimo.specs/geronimo-jms_1.1_spec/1.1: org.osgi.framework.BundleException: The provided URL is not a valid OSGi bundle. Consider using Karaf wrap feature, e.g. `install wrap:mvn:groupId/artifactId/version`. See more here: https://karaf.apache.org/manual/latest/#_wrap_deployer`